### ISSUE TITLE
Waveform often is significantly offset (~12ms) with MP3 files

### ISSUE THREAD

**Issue by Vulcanostrol** [wavesurfer-js/wavesurfer.js#2413]
Wavesurfer.js version: 5.2.0
Browser and operating system versions: Chrome on Windows 10 (default backend)

When using a waveform such as the following:
```
const wavesurfer = WaveSurfer.create({
    container: '#waveform-zoomed',
    waveColor: 'violet',
    progressColor: 'purple',
    minPxPerSec: 2000,
    scrollParent: true,
    plugins: [
        WaveSurfer.markers.create({
            markers: []
        })
    ]
})
```
...and using the `getCurrentTime()` function, it usually returns the correct position within the audio file:
![image](https://user-images.githubusercontent.com/4941629/143679977-897a4114-ec9b-406f-b72c-dab86e360ab6.png)

However, very often, there will be a ~12ms offset between what is shown by WaveSurfer and Audacity as shown below:
![image](https://user-images.githubusercontent.com/4941629/143679899-39de3f5d-3a57-432d-a29f-15e1d3d4a709.png)

I have tested around 20 different MP3 audio files, and around 15 of them had offsets (all around 11ms/12ms). I have also plotted the raw audio waveform using other languages and frameworks, such as in Python and R, and the Audacity waveforms, as shown above, are correct.

Looks like either `getCurrentTime()` is not exact (which I do not think is stated anywhere in the docs, plus it is weird that other waveforms are exact to the tenth of a millisecond), or the waveforms are not shown correctly, or WaveSurfer adds padding in front of audio when loading to do something (also something I am unaware of after looking through the docs and other issues).

If more info is needed, let me know 👍

**Comment by thijstriemstra** [comment]
Can you also try Firefox, and maybe other platforms, to see if that improves things?

**Comment by Vulcanostrol** [comment]
Hi thijs, its been a while since writing this issue. I have fixed the issue by using FFMPEG to encose the audio files in wav instead. The weird thing was: out of curiosity I tried encoding the files back to mp3 and the issue was gone. This means the issue is likely specific to a specific mp3 encoding.

I will try to recreate this error and then try the same files on different browsers and platforms
